src/main/kotlin/org/rust/lang/core/types/borrowck/ExprUseWalker.kt

```diff
@@ -421,23 +420,29 @@ class ExprUseWalker(private val delegate: Delegate, private val mc: MemoryCatego
             }
         }
 
-        if (withExpr == null) return
-
-        val withCmt = mc.processExpr(withExpr)
-        val withType = withCmt.ty
-        if (withType is TyAdt) {
-            val structFields = (withType.item as? RsStructItem)?.namedFields.orEmpty()
-            for (field in structFields) {
-                val isMentioned = fields.any { it.identifier?.text == field.identifier.text }
-                if (!isMentioned) {
-                    val interior = Interior.Field(withCmt, field.name)
-                    val fieldCmt = Cmt(withExpr, interior, withCmt.mutabilityCategory.inherit(), withType)
-                    delegateConsume(withExpr, fieldCmt)
+        // With base expr, e.g. `S { x, ..s }`
+        if (withExpr != null) {
+            val withCmt = mc.processExpr(withExpr)
+            val withType = withCmt.ty
+            if (withType is TyAdt) {
+                val withFields = (withType.item as? RsStructItem)?.namedFields.orEmpty()
+                for (withField in withFields) {
+                    val isMentioned = fields.any { it.identifier?.text == withField.identifier.text }
```

Comment:
```suggestion
                    val isMentioned = fields.any { it.name == withField.name }
```
... to support raw identifiers here.
Can be done in another PR b/c this behavior wasn't changed by the PR.

Reply:
Looks like it should be `val isMentioned = fields.any { it.referenceName == withField.name }` (`referenceName`, not `name`)